ideascube/views.py

```diff
@@ -25,23 +25,49 @@
 from ideascube.library.models import Book
 from ideascube.mediacenter.models import Document
 
+# Do not import directly Catalog as we want to mock it for test.
+from ideascube.serveradmin import catalog as catalog_mod
```

Comment:
Yes, you'd mock `ideascube.views.Catalog` instead, as we normally do.

But now I understand: because you do the mock in a fixture and not in the actual test, you can't know what you would need to mock, because that depends on the test using that fixture.

Hmmm....

Maybe that comment could be improved then, to make it completely obvious and avoidus breaking it in a future « refactoring » ?

Something explicitly saying that we need to be able to mock ideascube.serveradmin.Catalog.Catalog globally for all tests at once?